HACKING.md

```diff
@@ -0,0 +1,118 @@
+## Introduction
+LU is build with the [Entity–component–system](https://en.wikipedia.org/wiki/Entity_component_system) which you may know from the Unity Engine. In short that means everything that you see on screen (and more) is a GameObject on which Components can be attached. These components give the object functionality like rendering, physics and so on. 
```

Comment:
This is the wrong link I think, we don't use entity-component-system we use a entity-component system.

ECS contains an extra feature on top of EC that we and LU don't make use of.

Reply:
I'm a bit confused about that. I thought that this ECS is just an architecural pattern